lib/voight_kampff.rb

```diff
@@ -1,25 +1,23 @@
-require 'json'
+# frozen_string_literal: true
 
 require 'voight_kampff/test'
 require 'voight_kampff/methods'
 require 'voight_kampff/rack_request' if defined?(Rack::Request)
 require 'voight_kampff/engine' if defined?(Rails::Engine)
 
+# Class helper methods
 module VoightKampff
-  class << self
-    def root
-      require 'pathname'
-      Pathname.new File.expand_path '..', File.dirname(__FILE__)
-    end
+  ROOT = File.expand_path '..', __dir__
```

Comment:
I suggest to use brackets for more readability

Reply:
You're writing text in natural language (like English) mostly without parentheses, so "readability" is very subjective.